new/LIT6801KwellomuMalaekt.xml

```diff
@@ -0,0 +1,78 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6801KwellomuMalaekt" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ኵሎሙ፡ ሰራዊተ፡ መላእክት፡ መንፈሳዊያን፡ ነደ፡ እሳት፡ እሙንቱ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Kʷǝllomu sarāwita malāʾǝkt 
+                    nadda ʾǝsat ʾǝmmuntu...</title>
+                <title corresp="#t1" xml:lang="en">All hosts of spiritual angels, being flames of the fire...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2022-12-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6801KwellomuMalaekt" passive="SinaiEt004"/>
+                    <relation name="lawd:hasAttestation" active="LIT6801KwellomuMalaekt" passive="BAVet15"/>
+                    <relation name="saws:isDifferentTo" active="LIT4965Kwellomu" passive="BAVet15"/>
```

Comment:
I find the relations a bit confusing. The work is attested in two manuscripts, but is different from one of them? saws:isDifferentTo should not be used between a manuscript and a work record - can you find a better way to express the situation, possibly using `<desc>` inside the relation or describing the state in `<abstract>`?

Reply:
I think it should have been
`<relation name="saws:isDifferentTo" active="LIT6801KwellomuMalaekt" passive="LIT4965Kwellomu"/>`